_maps/map_files/IceBoxStation/IceBoxStation.dmm

```diff
@@ -5132,9 +5132,9 @@
 /obj/item/reagent_containers/spray/cleaner{
 	pixel_x = 2
 	},
-/obj/item/radio/headset/headset_medsci{
```

Comment:
Undocumented removal?

Reply:
Guilty. It was just there to fill out the room and I removed it and though absolutely nothing of it

Reply:
Geneticists can't actually get a medsci headset besides the one mapped in so you should leave it
(A few maps have one as a homage to the olden times)